eng/scripts/Language-Settings.ps1

```diff
@@ -3,6 +3,8 @@ $PackageRepository = "Maven"
 $packagePattern = "*.pom"
 $MetadataUri = "https://raw.githubusercontent.com/Azure/azure-sdk/master/_data/releases/latest/java-packages.csv"
 $BlobStorageUrl = "https://azuresdkdocs.blob.core.windows.net/%24web?restype=container&comp=list&prefix=java%2F&delimiter=%2F"
+$IndexHtmlLoc = "index.html"
+$AppTitle = "Azure SDK for Java"
```

Comment:
Should we just default this to "Azure SDK for $lang"?  If we do that and everything uses index.html by default then that will allow most languages to not have to pass any arguments to the UpdateDocIndexFiles call.

Reply:
.NET cannot apply this. 
$lang = "dotnet"
$AppTitle = "Azure SDK for NET"

Reply:
https://github.com/Azure/azure-sdk-tools/blob/master/eng/common/docgeneration/Generate-DocIndex.ps1#L152

Don't we have that already? Even if we need to pass in the ".NET" as the lang explicitly in that case we can still do that. However that is still better then needing to pass in "Azure SDK for" in every language.